src/lib/pure/ketrew_daemonize.mli

```diff
@@ -49,12 +49,18 @@
 
 (** The “standard” plugin-API. *)
 include Ketrew_long_running.LONG_RUNNING
+          with type run_parameters = Ketrew_gen_daemonize_v0.Run_parameters.t
 
 val create:
   ?starting_timeout:float ->
+  ?call_script:(string -> string list) ->
```

Comment:
I think I understand what `call_script` is doing, but only after looking at the implementation. I find it confusing.